examples/demuxer_isom.exs

```diff
@@ -39,7 +39,7 @@ defmodule Example do
     state = %{state | children_with_eos: MapSet.put(state.children_with_eos, element)}
 
     actions =
-      if Enum.all?([:sink_video, :sink_audio], &(&1 in state.children_with_eos)),
+      if Enum.all?([:sink_audio], &(&1 in state.children_with_eos)),
```

Comment:
why this?

Reply:
my oversight, was testing for audio only files and forgot to revert